core/services/ocr2/delegate.go

```diff
@@ -335,6 +331,11 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 				spec.RelayConfig["sendingKeys"] = []string{transmitterID}
 			}
 
+			chain, err2 := d.chainSet.Get(big.NewInt(chainID))
+			if err2 != nil {
+				return nil, errors.Wrap(err2, "ServicesForSpec failed to get chainset")
+			}
```

Comment:
Mercury only, so delay until necessary to unblock other products from using LOOP relayers.